mungegithub/github/github.go

```diff
@@ -1787,11 +1790,42 @@ func (obj *MungeObject) MergedAt() *time.Time {
 	return pr.MergedAt
 }
 
+func (config *Config) runMungeFunction(obj *MungeObject, fn MungeFunction) error {
+	if obj.Issue.Number == nil {
+		glog.Infof("Skipping issue with no number, very strange")
+		return nil
+	}
+	if obj.Issue.User == nil || obj.Issue.User.Login == nil {
+		glog.V(2).Infof("Skipping PR %d with no user info %#v.", *obj.Issue.Number, obj.Issue.User)
+		return nil
+	}
+	if *obj.Issue.Number < config.MinPRNumber {
+		glog.V(6).Infof("Dropping %d < %d", *obj.Issue.Number, config.MinPRNumber)
+		return nil
+	}
+	if *obj.Issue.Number > config.MaxPRNumber {
+		glog.V(6).Infof("Dropping %d > %d", *obj.Issue.Number, config.MaxPRNumber)
+		return nil
+	}
+	glog.V(2).Infof("----==== %d ====----", *obj.Issue.Number)
+	glog.V(8).Infof("Issue %d labels: %v isPR: %v", *obj.Issue.Number, obj.Issue.Labels, obj.Issue.PullRequestLinks != nil)
+	if err := fn(obj); err != nil {
+		return err
+	}
+	return nil
+}
+
 // ForEachIssueDo will run for each Issue in the project that matches:
 //   * pr.Number >= minPRNumber
 //   * pr.Number <= maxPRNumber
 func (config *Config) ForEachIssueDo(fn MungeFunction) error {
 	page := 1
+
+	// Using Zero time doesn't work with github
+	if config.since.IsZero() {
+		config.since = time.Date(1900, 1, 1, 0, 0, 0, 0, time.UTC)
+	}
+	since := time.Now()
```

Comment:
We may get some weirdness with clock skew here. Can we use the updated time of the last issue in the loop instead?